tests/sfdx-cli.test.ts

```diff
@@ -25,7 +24,7 @@ describe("CLI tests!", () => {
       }
 
       const scannerViolations = findings[0].violations;
-      expect(scannerViolations.length).toBe(6);
```

Comment:
Curious - why do we expect 2 less violations here?

Reply:
I hadn't run the tests in the initial change list for the TS conversion - there, I removed 2 violations. All the more reason to have CI run them! 🤦‍♂️